training_content/wiki_ed/slides/60-using-the-dashboard/6004-stage-2-create-in-the-sandbox.yml

```diff
@@ -0,0 +1,16 @@
+id: 6004
+title: "Stage 2: Create in the sandbox"
+summary:
+content: |
+  <figure>
+    <img src="https://upload.wikimedia.org/wikipedia/commons/6/62/Dashbooard.wikiedu.org_-_My_Articles_-_stage_2.png" />
+  </figure>
+
+  Once you've compiled your bibliography, it's time to draft your contribution. Follow the Sandbox link
```

Comment:
"Sandbox link" should likely have the same styling as the other links in being bolded.